tutorials/quickstarts/tracing_quickstart_openai.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>extra not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>